frontend/src/features/archived-items/crawl-list.ts

```diff
@@ -294,178 +158,104 @@ export class CrawlListItem extends LitElement {
- */
 @localized()
 @customElement("btrix-crawl-list")
-export class CrawlList extends LitElement {
-  static styles = [
-    srOnly,
-    rowCss,
-    columnCss,
-    hostVars,
-    css`
-      .listHeader,
-      .list {
-        margin-left: var(--row-offset);
-        margin-right: var(--row-offset);
-      }
-
-      .listHeader {
-        line-height: 1;
-      }
-
-      .list {
-        border: 1px solid var(--sl-panel-border-color);
-        border-radius: var(--sl-border-radius-medium);
-        overflow: hidden;
-      }
-
-      .row {
-        display: none;
-        font-size: var(--sl-font-size-x-small);
-        color: var(--sl-color-neutral-600);
-      }
-
-      .col {
-        padding-top: var(--sl-spacing-x-small);
-        padding-bottom: var(--sl-spacing-x-small);
-      }
-
-      @media only screen and (min-width: ${largeBreakpointCss}) {
-        .row {
-          display: grid;
-        }
-      }
-
-      ::slotted(btrix-crawl-list-item) {
-        display: block;
-        transition-property: background-color, box-shadow;
-        transition-timing-function: cubic-bezier(0.4, 0, 0.2, 1);
-        transition-duration: 150ms;
-      }
-
-      ::slotted(btrix-crawl-list-item:not(:first-of-type)) {
-        box-shadow: inset 0px 1px 0 var(--sl-panel-border-color);
-      }
-
-      ::slotted(btrix-crawl-list-item:hover),
-      ::slotted(btrix-crawl-list-item:focus),
-      ::slotted(btrix-crawl-list-item:focus-within) {
-        background-color: var(--sl-color-neutral-50);
-      }
-    `,
-  ];
+export class CrawlList extends TailwindElement {
+  static styles = css`
+    btrix-table {
+      --btrix-cell-gap: var(--sl-spacing-x-small);
+      --btrix-cell-padding-top: var(--sl-spacing-2x-small);
+      --btrix-cell-padding-bottom: var(--sl-spacing-2x-small);
+      --btrix-cell-padding-left: var(--sl-spacing-small);
+      --btrix-cell-padding-right: var(--sl-spacing-small);
+      --btrix-table-grid-auto-columns: min-content auto auto auto auto auto auto
+        min-content;
+    }
+  `;
 
   @property({ type: String })
   collectionId?: string;
 
   @property({ type: String })
   workflowId?: string;
 
-  @property({ type: String })
-  itemType: ArchivedItem["type"] = "crawl";
-
   @queryAssignedElements({ selector: "btrix-crawl-list-item" })
   listItems!: Array<HTMLElement>;
 
   render() {
-    return html` <div class="listHeader row">
-        <div class="col">
-          <slot name="idCol">${msg("Name")}</slot>
-        </div>
-        <div class="col">${msg("Date Created")}</div>
-        <div class="col">${msg("Size")}</div>
-        <div class="col">${msg("Created By")}</div>
-        <div class="col action">
-          <span class="srOnly">${msg("Actions")}</span>
-        </div>
-      </div>
-      <div class="list" role="list">
-        <slot @slotchange=${this.handleSlotchange}></slot>
-      </div>`;
+    return html` <div class="overflow-auto">
+      <btrix-table>
```

Comment:
If I'm reading the MDN articles right, I _think_ the more correct role here would be `grid` rather than the default `table` given that this is at least somewhat interactive, but it's not totally clear:

> An element with `role="table"` is a static tabular structure with rows containing cells. The cells are not focusable or selectable, though widgets within individual cells of the table can be interactive. Using a native HTML [`<table>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/table) element whenever possible is strongly encouraged.

— [ARIA: table role](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/table_role#:~:text=An%20element%20with%20role%3D%22table%22%20is%20a%20static%20tabular%20structure%20with%20rows%20containing%20cells.%20The%20cells%20are%20not%20focusable%20or%20selectable%2C%20though%20widgets%20within%20individual%20cells%20of%20the%20table%20can%20be%20interactive.%20Using%20a%20native%20HTML%20%3Ctable%3E%20element%20whenever%20possible%20is%20strongly%20encouraged.)

> A grid widget contains one or more rows with one or more cells of thematically related interactive content. While it does not imply a specific visual presentation, it implies a relationship among elements.

— [ARIA: grid role](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/grid_role#:~:text=A%20grid%20widget%20contains%20one%20or%20more%20rows%20with%20one%20or%20more%20cells%20of%20thematically%20related%20interactive%20content.%20While%20it%20does%20not%20imply%20a%20specific%20visual%20presentation%2C%20it%20implies%20a%20relationship%20among%20elements.)

Reply:
It should be table, since cells themselves are not interactive but the content inside maybe. Grid implies each cell can be focused and rotated through with the keyboard.

Reply:
Makes sense, thanks for the clarification!